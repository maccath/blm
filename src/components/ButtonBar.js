import React from "react"
import tw from "twin.macro"

const Container = tw.nav`
  flex space-x-5
`

export default function ButtonBar(props) {
  const children = props.children

  return (
    <Container className={props.className}>
      {React.Children.map(children, (child, i) => {
        return React.cloneElement(child, {
          className:
            "flex-initial rounded-lg bg-white text-purple p-3 no-underline",
        })
      })}
    </Container>
  )
}
