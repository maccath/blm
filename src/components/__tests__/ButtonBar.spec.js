import React from "react"
import renderer from "react-test-renderer"
import "jest-styled-components"

import ButtonBar from "../ButtonBar"

describe("Button Bar", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ButtonBar />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
