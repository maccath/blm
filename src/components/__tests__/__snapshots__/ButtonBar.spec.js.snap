// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Bar renders correctly 1`] = `
<nav
  className="css-1b6p5hc"
/>
`;
